### ISSUE TITLE
[feature request] Edit things with a right click or edit button instead of a double click

### ISSUE THREAD

**Issue by max-dmg** [manuelernestog/weektodo-web#151]
Currently, double clicking is the only way to edit a task name or the note for the task.

This is an accessibility issue, as some people can't double click fast enough for the program to pick up on it to edit the line, instead of just toggling the "done" status.

It would be nice to have one of the following:

- a button to edit each element;
- a right click menu to then click an edit option;
- middle click the element to be consistent with editing the task itself (not the best, because some people don't have a middle click).

Thanks.

**Comment by manuelernestog** [comment]
Hi @max-dmg sounds good to me :up: 
Thanks for the suggestion

**Comment by vic-t** [comment]
I agree that the double click to edit is neither intuitive (as it's really not used in many places elsewhere) nor accessibility-friendly. 

Personally, I'd prefer to keep things simple. A checkbox in front of the task to mark it done (it's a proven concept), a simple click on the title to open the task for further editing.

**Comment by SaifEl-Deen** [comment]
This is urgently needed, will make the whole working process a LOT easier.

**Comment by cooltnt** [comment]
Clicking Finish and double-clicking Edit causes the task to be marked as complete when the edit is completed, which is not expected in real time
